### DISCUSSION TITLE
[DONE] Adding Czech language

### DISCUSSION THREAD

**Discussion by kpetr01** [KL-Psychological-Methodology/ESMira#28] [Ideas]
Hi, is it possible to add the Czech language to Weblate? As a native Czech speaker, I would be more than happy to translate ESMira (once I am able to register - so far, I have not gotten the confirmation email; I've contacted the administrators).
Thank you very much.

**Comment by SelinaDev** [comment]
Hi! Thank you for your interest in translating ESMira! I have just added Czech as Language on Weblate.
That you are unable to register is odd, as it was working last week. Can you confirm that the email isn't in your spam folder?

Once things are working and you start to make progress with the languages feel free to contact us again, as we will need to manually update a few things to enable a new language (at least in the web part).

**Reply by kpetr01** [reply]
Hi, I think I am done with all the translations for the Czech language, so I am just letting you know. Thank you.

**Reply by SelinaDev** [reply]
Thank you very much for your help! I will make sure to include it in the web interface, and release it with the next update. It should also be included in the next app update. I'll have to evaluate whether I ship it alongside other changes or if this will be a standalone update. But it should be released in January at least.

Just a quick note. As we are always developing ESMira further we will often add new strings. So if you are interested in keeping the Czech version updated I would recommend to look into it again every few months.

**Reply by kpetr01** [reply]
Sure, thank you.

**Comment by JodliDev** [comment]
Hello, the contact form on weblate was misconfigured. So unfortuately your support email was never delivered. If you have any confidential information, you can use it (again) now. If not we can try to assist you in here.

But the general email service seems to be working. So your registration email should have been delivered. It usually takes a few minutes until emails are sent out. Maybe they ended up in your spam after you checked?

**Reply by kpetr01** [reply]
Hi, thank you for you assistance. I tried 3 e-mail, and the last one (private) finally worked, but warned me that redirecting to "jodli.dev" might be dangerous (I assume, that might be reason of the problems with other emails from gmail and my university). Thank you, I am good to go now.

**Reply by JodliDev** [reply]
Oh thank you for that feedback. That is very odd. I am assuming that warning was because the email came from another emailadress (other than jodli.dev)? Do you know which one? Because they all should come from jodli.dev.

If you can, could you send me a screenshot of that warning and also the source code of that email (in new Outlook: Right click email -> View message source; in Thunderbird: Open email -> More -> View Source). If you tell me what email software you are using, I can send you detailed instructions on how to get the source.

That would help me a lot to find out what went wrong

**Reply by kpetr01** [reply]
Yes, it came from noreply@jodli.dev. Since I have already confirmed the security of the link, the warning is not showing again. 

Unfortunately, my email client (seznam.cz) is showing only header source code, so I don't know if that helps (just the my real email is changed to myemail@email.com):

Received: from jodli.dev ([2a03:4000:2a:18f:7843:f7ff:fe1d:3b8d])
	by smtpd-mx-7cb6d7d4d8-v5647 (szn-email-smtpd/2.0.27) with ESMTP
	id 37c80b8e-1b4c-41cd-9c66-33b4d818f4d2;
	Wed, 11 Dec 2024 08:45:37 +0100
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/simple; d=jodli.dev; s=mail;
	t=1733903129; bh=BNa+NrS9qj8z+ODJQB/grypN8OkMSA8nnYDKY5JRK3M=;
	h=Subject:From:To:Date:From;
	b=qSfLL/jAvoxNaipjHl7GFJvJzoMHWE3owGmiSThfxWMIMz4DiEpbXN+QOx8x5MNfP
	 L40bYJZxV5rX6FiHHrY6gvHLfSNysFekWLE0WI57BMj+3nAzAm+CE7S/Zrm9tVSTQM
	 AM/EKi2DoFjEoNjXVqPSoaMWwMYxhyB5BiopoUxnHOzVNYy15Y5zszNjbjNL/YrIGA
	 MvnQuaG/NM4n9a0s5mPyBZ1W019jGTOQOKTtDpWkmtTR0G44B8qpDJIOsMkkwPKwW0
	 b6FujDiQuMHr8wk3CHgdJ5Id82a/kQK23MdLuxhtAJi3GwzLqZEXGfCFShdgraPfQT
	 JNN86+0qjNhBA==
Received: from 80eb0cdcb166 (unknown [172.18.0.4])
	by jodli.dev (Postfix) with ESMTP id 2D42164BC3D
	for <myemail@email.com>; Wed, 11 Dec 2024 08:45:29 +0100 (CET)
Content-Type: multipart/related;
 boundary="===============4478574547554973313=="
MIME-Version: 1.0
Subject: [Weblate] Your registration on Weblate
From: noreply@jodli.dev
To: myemail@email.com
Date: Wed, 11 Dec 2024 07:45:29 -0000
Message-ID: <173390312916.518.14613834864481592877@80eb0cdcb166>
X-Mailer: Weblate/5.6.2
X-Weblate-Notification: activation

Content-Type: multipart/alternative;
 boundary="===============7948569119228512078=="
MIME-Version: 1.0

Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit

Content-Type: text/html; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: quoted-printable

Content-Type: image/png
MIME-Version: 1.0
Content-Transfer-Encoding: base64
Content-ID: <email-logo.png@cid.weblate.org>
Content-Disposition: inline; filename="email-logo.png"

Content-Type: image/png
MIME-Version: 1.0
Content-Transfer-Encoding: base64
Content-ID: <email-logo-footer.png@cid.weblate.org>
Content-Disposition: inline; filename="email-logo-footer.png"

**Reply by JodliDev** [reply]
The header source is what I needed. Thank you very much!

From what I can see, everything seems to be in order. I have no explanation why your emails would not have gotten through or you would get a warning. I will keep an eye on it.

**Reply by JodliDev** [reply]
Ok I dove into some server logs and should have resolved the problem. Some providers seem to be weirdly specific with the format of ipv6 addresses for PTR records (translation: setting up email servers is a nightmare :D ). 

Thank you very much for pointing me to that issue!
